src/components/interface/Table/Table.js

```diff
@@ -132,7 +132,7 @@ const Table = ({
         <caption>{caption}</caption>
         {renderHeader()}
         <tbody>
-          {sortedData
+          {data.length > 0 && sortedData
```

Comment:
I don't understand why we need this ? 
If data.length === 0 then sortedData is empty, so the .map returns nothing ?